handlers/relcal/relcalhandler.go

```diff
@@ -0,0 +1,20 @@
+package relcal
+
+import (
+	"net/http"
+)
+
+func Handler(useNewReleaseCalendar bool, newHandler, oldHandler http.Handler) http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		serve(w, r, newHandler, oldHandler, useNewReleaseCalendar)
+	})
+}
+
+func serve(w http.ResponseWriter, req *http.Request, n, o http.Handler, useNewReleaseCalendar bool) {
```

Comment:
```suggestion
func serve(w http.ResponseWriter, req *http.Request, newHandler, oldHandler http.Handler, useNewReleaseCalendar bool) {
```

clearer var names please